src/templates/InstantSearch.js/src/app.js.hbs

```diff
@@ -34,24 +31,28 @@ search.addWidget(
   {{/each}}
 </article>
 `,
-    },
-    {{/if}}
-  })
-);
+  },
+  {{/if}}
+});
 
 {{#each attributesForFaceting}}
-search.addWidget(
-  instantsearch.widgets.refinementList({
-    container: '#{{this}}-list',
-    attribute: '{{this}}',
-  })
-);
+const {{this}}RefinementList = instantsearch.widgets.refinementList({
```

Comment:
what if the attribute can't be used as a variable? like `my-thing`? We might need to camelcase it, or directly add them in the tree without giving names.

Reply:
I thought about attributes starting with numbers (not likely) but not about that indeed. I'll camelcase those.